frontend/public/index.html

```diff
@@ -8,7 +8,7 @@
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
 
-  <body>
+  <body class="bg-dark">
```

Comment:
Maybe we should keep the light background so that the page isn't so _heavy_, what do you think ?

Reply:
I agree, lets not dampen the mood ;)